package.json

```diff
@@ -16,35 +16,50 @@
     "node": ">=12.16.1"
   },
   "scripts": {
-    "dev": "lerna run start --scope @blitzjs/* --stream --parallel",
     "postinstall": "npm rebuild husky",
-    "build": "lerna run build --scope @blitzjs/* --stream",
-    "prepublishOnly": "lerna run build --scope @blitzjs/* && lerna run test --parallel",
-    "pretest": "lerna run build --scope @blitzjs/*",
-    "test": "lerna run test --parallel"
+    "dev": "lerna run dev --no-private --stream --parallel",
+    "link-cli": "cd packages/blitz && yarn link",
+    "unlink-cli": "cd packages/blitz && yarn unlink",
+    "build": "lerna run build --no-private --stream",
+    "test": "lerna run test --parallel",
+    "prepublishOnly": "yarn run build && yarn run test"
   },
   "devDependencies": {
     "@types/debug": "^4.1.5",
     "@types/jest": "^25.1.3",
     "@types/node": "^13.7.4",
+    "@wessberg/cjs-to-esm-transformer": "^0.0.19",
+    "@wessberg/rollup-plugin-ts": "^1.2.24",
+    "chalk": "^4.0.0",
+    "concurrently": "^5.1.0",
     "cross-env": "^7.0.0",
     "debug": "^4.1.1",
     "husky": "^4.2.3",
     "jest": "24.9.0",
     "lerna": "^3.20.2",
     "lint-staged": "^10.0.8",
+    "minimist": "^1.2.5",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.4",
     "pretty-quick": "2.0.1",
+    "prompt": "^1.0.0",
+    "rollup": "^2.6.1",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-json": "^4.0.0",
+    "rollup-plugin-node-polyfills": "^0.2.1",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-peer-deps-external": "^2.2.2",
+    "semver": "^7.3.2",
```

Comment:
Out of interest/curiosity why are we installing everything in the root? Was workspaces screwing up stuff?

Reply:
I'm not sure. @medelman17 did this. Why in the root vs in the package Micheal?

Reply:
He says all dev dependencies should be installed at root so that everything absolutely stays in sync and uses the same version. Makes sense to me.